portal/src/main/scala/cromwell/pipeline/datastorage/DatastorageModule.scala

```diff
@@ -6,8 +6,10 @@ import cromwell.pipeline.database.PipelineDatabaseEngine
 import cromwell.pipeline.datastorage.dao.entry.UserEntry
 import cromwell.pipeline.datastorage.dao.repository.UserRepository
 
+import scala.concurrent.ExecutionContext
+
 @Module
-class DatastorageModule(config: Config) {
+class DatastorageModule(config: Config)  (implicit executionContext: ExecutionContext){
```

Comment:
Do we need `executionContext` here?

Reply:
If we remove `executionContext` from the `UserRepository` which is excess there then we can remove it here too.